### PR TITLE
Indices PUT Mapping API docs: Remove mapping type user and rephrase first sentence

### PR DESCRIPTION
**Pull request by Joostrothweiler**
The current docs on [Indices APIs: PUT Mapping](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html) suggests that a having number of different mapping types per index is still possible in elasticsearch versions > 6.0.0 although they have been [removed](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html). The console code has already been updated accordingly but notes (2) and (3) on the console code still name the `user` mapping type.

This PR updates the list with notes after the console code, as well as the first sentence of the docs to avoid confusion. Also, I have removed the second command from the console code as it no longer holds any value if the docs are solely on the `_doc` mapping.

### CODE REVIEW

File: docs/reference/indices/put-mapping.asciidoc [hunk 1]
```diff
@@ -1,24 +1,14 @@
 [[indices-put-mapping]]
 == Put Mapping
 
-The PUT mapping API allows you to add a new type to an existing index, or add new
-fields to an existing type:
+The PUT mapping API allows you to define a mapping for a new or existing index by adding fields to your mapping type.
```
Comment (bleskes) [inline docs/reference/indices/put-mapping.asciidoc]: good catch, but a small nit - the put mapping api only works for existing indices. How about `The PUT mapping API allows you to add fields to an existing index or to change search only settings of existing fields`?

**Review by bleskes — CHANGES_REQUESTED** [review]
Thx @Joostrothweiler . I left a small comment

File: docs/reference/indices/put-mapping.asciidoc (hunk 1 above)
Comment (Joostrothweiler) [reply docs/reference/indices/put-mapping.asciidoc]: Ah makes sense! Just made the change.

**Comment by Joostrothweiler** [comment]
Done!

**Review by bleskes — APPROVED** [review]
Thx @Joostrothweiler

**Pull request merged**